release-notes/aws/v9.0.1.md

```diff
@@ -33,4 +33,11 @@ Below, you can find more details on components that were changed with this relea
 
 ### coredns v1.6.5 ([Giant Swarm app v1.1.7](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v117-2020-03-19))
 
+- Updated coredns to upstream version 1.6.5.
+- Removed CPU limits.
+- Updated manifests for Kubernetes 1.16.
+- Fix HPA manifests for Kubernetes 1.16.
```

Comment:
Good call. I deleted the `Fix HPA` line as well as the `Change autopath to disable` since it is covered in "Make `autopath` plugin configurable, optional and disabled by default."